inception-support/src/main/java/de/tudarmstadt/ukp/inception/support/annotation/OffsetSpan.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2019
+ * Ubiquitous Knowledge Processing (UKP) Lab
+ * Technische Universität Darmstadt
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package de.tudarmstadt.ukp.inception.support.annotation;
+
+import java.io.Serializable;
+
+public class OffsetSpan implements Serializable
+{
+
+    private static final long serialVersionUID = -2532025576874280981L;
+
+    private int start;
+
+    private int endExclusive;
+
+    public OffsetSpan(int aStart, int aEndExclusive) {
+        start = aStart;
+        endExclusive = aEndExclusive;
+    }
+
+    public int getStartOffset()
+    {
+        return start;
+    }
+
+    public int getEndOffsetExclusive()
```

Comment:
We usually call these `getEnd()`. Ranges in UIMA are always half-open `[begin,end)`. If you want to make that explicit, I'd suggest you add it as JavaDoc instead of using a verbose name.

Reply:
Renamed.